### DISCUSSION TITLE
Cannot access my GitHub Education pack.

### DISCUSSION THREAD

**Discussion by 4reha** [community/community#36006] [GitHub Education]
Hello There, I Hope you're all doing well.
I have a problem recently, I lost my access to the GitHub Education pack and my PRO badge too. 
I'm studying at GitHub Campus partner school, and I gain my access from a school administrator, And I tried to sign up for the Student Developer pack again but it display this message:  "We're so glad you're here! Thank you, again, for joining."  and keeps redirecting me to these ([1](https://education.github.com/), [2](https://education.github.com/pack/join), [3](https://education.github.com/pack)) same pages.

Any help would be appreciated, thannks

**Comment by forgerlil** [comment]
I posted the exact same question here, shortly before you, still waiting the answer: https://github.com/orgs/community/discussions/36002
:D

**Comment by samus-aran** [comment]
Hi @4reha & @forgerlil 👋 
Are you both from the same school? Did you both get a verification email to confirm that your school's coupon was applied?

**Reply by 4reha** [reply]
I guess we are not from the same school. and yes I have got a verification email. and I have been using  StudentDeveloperPack  for  more than a year 
<img width="389" alt="Screen Shot 2022-10-13 at 11 16 44 AM" src="https://user-images.githubusercontent.com/80625058/195571204-9e0fa347-883e-4e20-9bc7-97f074b4dffd.png">

**Reply by forgerlil** [reply]
Hey! I assume we are not from the same school, either. I got an e-mail: 
![image](https://user-images.githubusercontent.com/81853712/195587410-3095c6d2-a10a-4063-8154-53bc0e2a2683.png)

I still get redirected to <code>https://education.github.com/pack/join</code> whenever I click on <code>https://education.github.com/pack/offers</code> :(
